test/utils.ts

```diff
@@ -4,7 +4,13 @@ import type { ExpressionTestEvaluation } from './ExpressionFixtures/base';
 import { baseFixtures } from './ExpressionFixtures/base';
 
 export const testExpressionsWithEvaluator = (Evaluator: ExpressionEvaluatorClass) => {
-	const tourn = new Tournament(() => {}, undefined, Evaluator);
+	const tourn = new Tournament(
+		(e) => {
+			console.error(e);
+		},
+		undefined,
+		Evaluator,
```

Comment:
Nothing to address now but maybe the args would be nicer as an object in future.